frontend/dist/shotgrid-addon.js

```diff
@@ -129,14 +129,19 @@ const syncUsers = async () => {
   sgUsers = await getShotgridUsers();
 
   sgUsers.forEach((sg_user) => {
+    // Make sure login string is normalized and avoid all special characters with underscore
```

Comment:
which users did you have in SG that caused this to fail? we should add some examples

Reply:
in our testing server all users are having shotgrid user login filled as their email. So it was failing an all users with `name@domain`

Reply:
ohh I see that's fair, we should then just grab everything except the domain instead of replacing special characters?